docs/features/vision/fog_of_war_manager.md

```
@@ -0,0 +1,433 @@
+
+const DEFAULT_SIZE : Vector2i = Vector2i(100, 100)
+
+## Color of area the units saw and not seeing currenty. Can see in Debug Texture View
+@export var fog_circle_color : Color = Color(0.25, 0.25, 0.25)
+## Color of area the units see around them. Can see in Debug Texture View
+@export var shroud_circle_color : Color = Color(1.0, 1.0, 1.0)
+@export_category("Fog Values")
+
+## The size of a single pixel in the 3D world
+@export_range(1, 10000, 1,"suffix:px/length") var texture_units_per_world_unit : int = 2 : # px/length
+		set(value):
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)